.gitignore

```diff
@@ -1,2 +1,3 @@
 /target/
 **/*.rs.bk
+.idea/
```

Comment:
FYI, changes like this are better placed in your `.gitconfig`'s `[core] excludesfile`, so that our `.gitignore` files don't turn into a giant list of editor-specific errata which needs to be maintained on a repo-by-repo basis.

Saying this as a fellow IntelliJ user.

Reply:
That's a nice feature. Thanks for letting me know.